Source/SwiftLintFramework/Extensions/File+Cache.swift

```diff
@@ -88,7 +88,7 @@ extension File {
 
     internal var assertHandler: (() -> ())? {
         get {
-            return assertHandlers[cacheKey] ?? nil
+            return assertHandlers[cacheKey]
```

Comment:
Is this change safe? Aren't there cases where we put `nil` as a valid value in `assertHandlers`?